src/interfaces/http/server.js

```diff
@@ -15,6 +15,6 @@ module.exports = ({
     start: () => express
       .disable('x-powered-by')
```

Comment:
remove this line, please, as `helmet` already disables the `powered-by` by default;